scripts/pysa_fuzzer/Makefile

```diff
@@ -0,0 +1,31 @@
+# Generate Python files
+generate:
+	python generate_code.py
+
+# Set up Python virtual environment and install dependencies
+setup:
+	python3 -m venv tutorial
+	. tutorial/bin/activate && \
+	pip3 install pyre-check fb-sapp django-stubs
```

Comment:
Can we just assume the user installed pyre? It seems wasteful to re-install pyre everything I want to run that script.
Also, we shouldn't need fb-sapp and django-stubs here.